### PR TITLE
feat(Lib): print the disassembled info

### PR DESCRIPTION
**Pull request by tai271828**
This pull resolves Issue: #2889 

**Steps to Test This Pull Request**
```
$ target/debug/rustpython -m dis ./demo.py

                 0 LoadNameAny          (0, print)
                 1 LoadConst            ("Hello, RustPython!")
                 2 CallFunctionPositional (1)
                 3 Pop
                 4 LoadConst            (None)
                 5 ReturnValue
```

### CODE REVIEW

**Comment by tai271828** [comment]
Maybe also related to issue #367

**Review by youknowone — APPROVED** [review]
thank you for contributing!

**Comment by youknowone** [comment]
Hmm, something is broken

File: Lib/dis.py [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+
+# Disassembling a file by following cpython Lib/dis.py
+parser = argparse.ArgumentParser()
+parser.add_argument('infile', type=argparse.FileType('rb'), nargs='?', default='-')
+args = parser.parse_args()
+with args.infile as infile:
+    source = infile.read()
+code = compile(source, args.infile.name, "exec")
+_dis.dis(code)
```
Comment (youknowone) [inline Lib/dis.py:12]: This file is not only used for the command `rustpyhon -m dis` but also used as an independent module like `import dis` in common python code.
So this new codes must be isolated by guarding `if __name__ == "__main__":` just like CPython handle it.

**Review by youknowone — CHANGES_REQUESTED** [review]

File: Lib/dis.py [hunk 2]
```diff
@@ -1 +1,19 @@
-from _dis import *
```
Comment (coolreader18) [inline Lib/dis.py:1]: I think you should keep this line, and then simply call `dis()` in `_test`

File: Lib/dis.py [hunk 3]
```diff
@@ -1 +1,19 @@
...
+import argparse
```
Comment (coolreader18) [inline Lib/dis.py:1]: Since there's also an `import argparse` inside the function, this can be removed

Comment (tai271828) [reply Lib/dis.py:1]: Oooops, right. Thanks for the good catch!

**Comment by tai271828** [comment]
Thanks @coolreader18 's review. Re-based and re-submitted. This pull request is ready to review again.

**Review by coolreader18 — APPROVED** [review]
LGTM, thank you for contributing!

**Pull request merged**